portal/views/portal.py

```diff
@@ -665,22 +665,66 @@ def profile(user_id):
 
 @portal.route('/privacy')
 def privacy():
-    """ privacy use page, ONLY in Truenth, Not Eproms"""
-    if current_app.config.get('GIL'):
-        privacy_resource = VersionedResource(app_text(PrivacyATMA.name_key()))
-        return render_template(
-            'gil/privacy.html',
-            content=privacy_resource.asset, user=current_user(),
-            editorUrl=privacy_resource.editor_url)
+    """ privacy use page"""
+    gil = current_app.config.get('GIL')
+    user = current_user()
+    if user:
+        def get_top_org(user):
```

Comment:
This doesn't feel like the right location for such a method.  We have an OrgTree that could probably better support it.

Furthermore, it doesn't look to consider more than the user's first organization?